bigquery/storage_client.go

```diff
@@ -135,13 +135,19 @@ type readSession struct {
 
 // Start initiates a read session
 func (rs *readSession) start() error {
+	preferredMinStreamCount := int32(0)
```

Comment:
this looks a bit odd, why not just `var preferredMinStreamCount int32`?

Reply:
good point, let me change here. I think I was playing with default values and ended up leaving like that.